tasks/ThrusterTask.cpp

```diff
@@ -95,14 +93,13 @@ void ThrusterTask::cleanupHook()
     node->Fini();
 }
 
-void ThrusterTask::setGazeboModel(WorldPtr _world,  ModelPtr _model, sdf::ElementPtr _plugin)
+void ThrusterTask::setGazeboThruster( ModelPtr model )
```

Comment:
Even if we define tasks in different packages... we will still need to instantiate them inside the rock-gazebo system plugin ? If not, how they will be instantiated ?